tpr/kind_test.go

```diff
@@ -0,0 +1,43 @@
+package tpr
+
+import (
+	"testing"
+
+	"github.com/juju/errgo"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractKindAndGroup(t *testing.T) {
+	tests := []struct {
+		Input         string
```

Comment:
Think this should be called `name`, to reflect the actual argument the function takes.